deploy/charts/arcadia/templates/kubebb-menus.yaml

```diff
@@ -0,0 +1,357 @@
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: Menu
+metadata:
+  creationTimestamp: null
```

Comment:
should we move this `kubebb-menues.yaml` and `kubebb-portal.yaml` inside `portal.yaml` ?

Reply:
that's make it separate for now